server.js

```diff
@@ -57,7 +62,20 @@ app.use(require('./middleware/load-options'));
 app.post('/', require('./routes/post'));
 app.get('/*', require('./routes/get'));
 
-// Start listening for HTTP requests
-app.listen(PORT, () => {
-  console.log(`🚀 on http://localhost:${PORT}`);
-});
+// If the current process is the main one
+if (cluster.isMaster) {
```

Comment:
Can you revert this change. If we want to merge this code, we need to do it in small succinct parts. I don't want to add multi-core it adds more plates and the production instance is single core in the first place.

Reply:
@remy I've rolled it back.